README.md

```diff
@@ -63,7 +63,8 @@ class NATS {
 	int subscribe(const char* subject, sub_cb cb, const char* queue = NULL, const int max_wanted = 0);
 	void unsubscribe(const int sid);
 
-	int request(const char* subject, const char* msg, sub_cb cb, const int max_wanted = 1);
+	char* generate_inbox_subject();
```

Comment:
`generate_inbox_subject` is a private method, I'd rather leave it internal anyway since the returned buffer needs to be freed be the caller. It's not really needed if there's an overload for the original `request` functionality.